concrete_datastore/settings/base.py

```diff
@@ -478,3 +478,4 @@
     "TemporaryToken",
     "SecureConnectToken",
 ]
+ADMIN_URL_ENABLED = True
```

Comment:
Based on my previous comment, I would add

```suggestion
ADMIN_URL_ENABLED = True
ADMIN_ROOT_URI = "concrete-datastore-admin"
```